dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassReferenceGraphTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static com.google.cloud.tools.opensource.classpath.ClassDumperTest.GRPC_CLOUD_FIRESTORE_JAR;
+import static com.google.cloud.tools.opensource.classpath.ClassDumperTest.absolutePathOfResource;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.truth.Truth;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.junit.Test;
+
+public class ClassReferenceGraphTest {
+  private static ClassReferenceGraph createExampleGraph() throws URISyntaxException, IOException {
+    ClassSymbolReference grpcToA =
```

Comment:
one line

Reply:
Added an empty line.

Reply:
I'm not sure why we need the empty line. And if google-java-format formats it this way, we should accept that.